contrib/gorilla/mux/mux_test.go

```diff
@@ -11,70 +13,76 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-func TestHttpTracer200(t *testing.T) {
-	assert := assert.New(t)
-	mt := mocktracer.Start()
-	defer mt.Stop()
+var httpTests = []struct {
```

Comment:
This is good, but since we are not using this variable anywhere else, we should just inline it, like in [this example](https://github.com/DataDog/dd-trace-go/blob/gbbr/priority/contrib/database/sql/internal/dsn_test.go#L11). That way we use the same style throughout the repository.